### PR TITLE
add row for actual gas cost in go

### PR DESCRIPTION
**Pull request by chovy**
refs #33

### CODE REVIEW

File: routes/web3relay.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ exports.data = function (req, res) {
       } else {
         var ttx = tx;
         ttx.value = etherUnits.toEther(new BigNumber(tx.value), "wei");
+        ttx.actualGasCost = web3.fromWei(tx.gasPrice * tx.gas , 'ether');
```
Comment (jmank88) [inline routes/web3relay.js]: Is it safe to mulitply these? Or do we need to carefully construct a `BigNumber` first?

Comment (chovy) [reply routes/web3relay.js]: I'm not sure, that's what @treeder told me to do.

Comment (jmank88) [reply routes/web3relay.js]: Typical/minimum values for `gas` and `gasPrice` are  `21000` and `2000000000000000000`, so the result of this multiplication will be `>=4.2e+22` which would overflow a `uint64`. I'm not sure what javascript does here, but overflowing or falling back to floating point are both no good. You should be able to convert both values to `BigNumber` to do the multiplication safely, then call `etherUnits.toEther` with the resulting `BigNumber` just like the previous line.

Comment (jmank88) [reply routes/web3relay.js]: How would the math work? Javascript doesn't have arbitrary precision arithmetic built-in, does it?

I think that maintaining full precision during the computation (with `BigNumber`), and displaying it formatted via the `bigNumber` function are two separate things that both need to be done.

Comment (chovy) [reply routes/web3relay.js]: ok

**Comment by chovy** [comment]
test please

**Pull request merged**